config/jobs/kubernetes/sig-node/containerd.yaml

```diff
@@ -134,6 +134,7 @@ periodics:
       - --scenario=kubernetes_e2e
       - --
       - --check-leaked-resources
+      - --env=KUBE_GCI_VERSION=cos-85-13310-1308-1
```

Comment:
hmm, will the also affect the node image? I think we still want the node image to be ubuntu, but have the master image be older version of COS?

I'm not sure behavior of passing in `KUBE_GCI_VERSION` and `--gcp-node-image=ubuntu` via kubetest...

Reply:
If we just want to change the master image back to older COS, I think we can just change `KUBE_GCE_MASTER_IMAGE` and `KUBE_GCE_MASTER_PROJECT`

Reply:
Done!